ui/components/Shared/SharedSlideUpMenu.tsx

```diff
@@ -4,24 +4,27 @@ interface Props {
   isOpen: boolean
   close: () => void
   children: React.ReactNode
-  size: "small" | "medium" | "large"
+  customSize?: string
+  size: "small" | "medium" | "large" | "custom"
 }
 
 export default function SharedSlideUpMenu(props: Props): ReactElement {
-  const { isOpen, close, size, children } = props
+  const { isOpen, close, size, children, customSize } = props
 
   let menuHeight = "536px"
   if (size === "large") {
     menuHeight = "600px"
   } else if (size === "small") {
     menuHeight = "268px"
+  } else if (size === "custom") {
+    menuHeight = customSize || "600px"
   }
 
   return (
     <div
-      className={`slide_up_menu
-        ${size === "large" ? " large" : ""}
-        ${!isOpen ? " closed" : ""}`}
+      className={`slide_up_menu ${size === "large" ? " large" : ""}${
+        !isOpen ? " closed" : ""
+      }`}
```

Comment:
Let's use `classNames` here:
```suggestion
      className={classNames("slide_up_menu", { large: size === "large", closed: !isOpen })`}
```

(Plus run `prettier` :grin:)

Reply:
Oh, neat, I was not using `classNames` before!